README.md

```diff
@@ -683,6 +683,21 @@ make run
 ```
 This is mostly useful for local development.
 
+### Note on namespace removal
+Many custom resources deployed with the compliance operators use finalizers
+to handle dependencies between objects. If the whole operator namespace gets
+deleted (e.g. with `oc delete ns/openshift-operator`), the order of deleting
```

Comment:
was `ns/openshift-operator` intentional?

Reply:
No, that was a typo thanks..

(Hello, my name is Jakub, I'm a Senior Software Engineer and I have trouble patching a README :-) ...)